.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2.1
 orbs:
   git-shallow-clone: guitarrapc/git-shallow-clone@2.8.0
 
-efcms-docker-image: &efcms-docker-image $AWS_ACCOUNT_ID.dkr.ecr.us-east-1.amazonaws.com/ef-cms-us-east-1:4.2.0
+efcms-docker-image: &efcms-docker-image $AWS_ACCOUNT_ID.dkr.ecr.us-east-1.amazonaws.com/ef-cms-us-east-1:4.3.0
```

Comment:
Shouldn't this be 4.2.1? The updates in the dockerfile don't look substantial enough to warrant incrementing the minor version.

Reply:
Discussed with Jim...were going to leave the version as is.